prompting/llms/vllm_llm.py

```diff
@@ -154,14 +154,14 @@ def _make_prompt(self, messages: List[Dict[str, str]]):
 
         for message in messages:
             if message["role"] == "system":
-                composed_prompt += f'<|system|>{message["content"]} '
+                composed_prompt += f'<|im_start|>system\n{message["content"]} <|im_end|>'
             elif message["role"] == "user":
-                composed_prompt += f'<|user|>{message["content"]} '
+                composed_prompt += f'<|im_start|>user\n{message["content"]} <|im_end|>'
             elif message["role"] == "assistant":
-                composed_prompt += f'<|assistant|>{message["content"]} '
+                composed_prompt += f'<|im_start|>assistant\n{message["content"]} <|im_end|>'
```

Comment:
Can you add a comment on the code stating the model being used and the chat template url?

example:
```python
# Chat template from upstage/SOLAR-10.7B-Instruct-v1.0,
```

If you could also add the url of the chat template in the description of the PR it would be good for future reference

Reply:
https://huggingface.co/NousResearch/Nous-Hermes-2-SOLAR-10.7B